pkg/releaseutil/manifest_sorter.go

```diff
@@ -53,16 +52,16 @@ type result struct {
 // TODO: Refactor this out. It's here because naming conventions were not followed through.
 // So fix the Test hook names and then remove this.
 var events = map[string]release.HookEvent{
-	hooks.PreInstall:         release.HookPreInstall,
-	hooks.PostInstall:        release.HookPostInstall,
-	hooks.PreDelete:          release.HookPreDelete,
-	hooks.PostDelete:         release.HookPostDelete,
-	hooks.PreUpgrade:         release.HookPreUpgrade,
-	hooks.PostUpgrade:        release.HookPostUpgrade,
-	hooks.PreRollback:        release.HookPreRollback,
-	hooks.PostRollback:       release.HookPostRollback,
-	hooks.ReleaseTestSuccess: release.HookReleaseTestSuccess,
-	hooks.ReleaseTestFailure: release.HookReleaseTestFailure,
+	release.HookPreInstall.String():   release.HookPreInstall,
+	release.HookPostInstall.String():  release.HookPostInstall,
+	release.HookPreDelete.String():    release.HookPreDelete,
+	release.HookPostDelete.String():   release.HookPostDelete,
+	release.HookPreUpgrade.String():   release.HookPreUpgrade,
+	release.HookPostUpgrade.String():  release.HookPostUpgrade,
+	release.HookPreRollback.String():  release.HookPreRollback,
+	release.HookPostRollback.String(): release.HookPostRollback,
+	release.HookTest.String():         release.HookTest,
+	"test-success":                    release.HookTest,
```

Comment:
Might be good to put his string in a variable or constant and also leave a comment here explaining this is a for legacy support and should be removed later.

Reply:
Leaving as a string literal for now -- I don't want to encourage anyone to reference this since at the moment this is the only place we need to use the value.